src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -54,17 +54,22 @@ export class CalcitePickListItem {
    */
   @Prop() metadata?: object;
 
-  @Watch("metadata") metadataWatchHandler() {
+  @Watch("metadata") metadataWatchHandler(): void {
     this.calciteListItemPropsChange.emit();
   }
 
+  /**
+   * Set this to true to display a remove action that removes the item from the list.
+   */
+  @Prop({ reflect: true }) removable? = false;
```

Comment:
I think this shoud be `@Prop({ reflect: true }) removable?: boolean;`

Reply:
This wouldn't be correct though since the default value would be `undefined` when querying the prop on the instance.

Reply:
hmm then I don't think the `?` will do anything here. It only seems to work when a prop is undefined at the get go.

Reply:
We can tackle that separately. I think there are other optional props that will need updating.